### ISSUE TITLE
Support Material You Theming with App Icon

### ISSUE THREAD

**Issue by matzex** [PhilippC/keepass2android#2070]
Hello,

I am a long-term user of the keepass2android app and like it for years!
It would be nice if you could support Material You Theming for the App Icon which was introduced with Android 13 and should be easy to implement with a monochrom icon.
Spotify just released their monochrome icon: https://community.spotify.com/t5/Live-Ideas/Android-Other-Modernize-app-icon-to-support-Android-13-Material/idi-p/5424376

Maybe this helps to implement it (but I did not test it): https://proandroiddev.com/implement-themed-icons-android-13-d20b89233681

Best regards
Matthias

**Comment by waaweech** [comment]
Agree. This app needs an update, especially for Android 13. The last update was a year ago. Hopefully the developer is still maintaining it.
